bumble/device.py

```diff
@@ -3533,7 +3533,11 @@ def add_default_services(self, generic_access_service=True):
     async def notify_subscriber(self, connection, attribute, value=None, force=False):
         await self.gatt_server.notify_subscriber(connection, attribute, value, force)
 
-    async def notify_subscribers(self, attribute, value=None, force=False):
+    async def notify_subscribers(self,
+        attribute: Attribute,
+        value: Optional[bytes] = None,
+        force: bool = False,
+    ) -> None:
```

Comment:
Added type annotations based on poking around at the callers, but since of course the type info isn't pervasive I'm not sure these are right

Reply:
This is good. We want to eventually have everything type, incrementally.